### PR TITLE
Feature/merging checkout for live

### PR DESCRIPTION
**Pull request by Affan007**

### CODE REVIEW

File: theme/assets/surprisebox-survey-checkout-td2.js [hunk 1]
```diff
@@ -0,0 +1,939 @@
+let billingAddress, shippingAddress, klarnaObjectIs;
+$(function() {
+  loadCartProd();
+  cartProdTable();
+  hideElements();
+});
+
+const hideElements = () => {
```
Comment (PygmalionPolymorph) [inline theme/assets/surprisebox-survey-checkout-td2.js]: Can you remove this?

File: theme/assets/surprisebox-survey-checkout-td2.js [hunk 2]
```diff
@@ -0,0 +1,939 @@
...
+  })
+    .then(resp => {
+      if (resp.status === 200) {
+        return resp.json();
+      } else {
+        return Promise.reject("");
+      }
+    })
+    .then(resp => {
+      console.log("main console ", resp);
```
Comment (PygmalionPolymorph) [inline theme/assets/surprisebox-survey-checkout-td2.js]: Please remove all console.logs

File: theme/assets/surprisebox-survey-checkout.js [hunk 3]
```diff
@@ -113,18 +114,23 @@ function loadCartProd() {
...
       if (resp.status === 200) {
-        console.log("@@@@@@@@@@@@@ 200 @@@@@@@ ");
         return resp.json();
       } else {
         return Promise.reject("");
       }
     })
     .then(resp => {
       console.log("main console ", resp);
+      // if(resp.order_details.financial_status === "paid"){
```
Comment (PygmalionPolymorph) [inline theme/assets/surprisebox-survey-checkout.js]: Remove

File: theme/assets/surprisebox-survey-checkout.js [hunk 4]
```diff
@@ -113,18 +114,23 @@ function loadCartProd() {
...
+      //   //order is paid. So showing the pink screen
+      //   confirmationPaymentIs();
+      //   $("body .loader").remove();
+      // }
+      // else{
       let checkFullFilment = resp.order_details.fulfillment_status;
       let checkingFulfilmentLen = resp.order_details.fulfillments.length;
       let checkDates = false;
-
+      debugger;
```
Comment (PygmalionPolymorph) [inline theme/assets/surprisebox-survey-checkout.js]: Remove

File: theme/assets/surprisebox-survey-checkout.js [hunk 5]
```diff
@@ -337,7 +343,9 @@ function loadCartProd() {
       currency : ${resp.order_details.currency}
       Total Price : ${resp.order_details.total_price}
     `);
-      console.log(resp);
+      //console.log(resp);
```
Comment (PygmalionPolymorph) [inline theme/assets/surprisebox-survey-checkout.js]: Remove

File: theme/templates/page.surprisebox-survey-checkout.liquid [hunk 6]
```diff
@@ -14,7 +14,7 @@
   <div class="backdrop">
     <span style="color: rgb(221, 0, 0);">
       <span>Es wird geladen...</span>
-      <i class="close fas fa-times-circle"></i>
+      {% comment %} <i class="close fas fa-times-circle"></i> {% endcomment %}
```
Comment (PygmalionPolymorph) [inline theme/templates/page.surprisebox-survey-checkout.liquid]: Remove

File: theme/assets/surprisebox-survey-checkout.js (hunk 3 above)
Comment (Affan007) [reply theme/assets/surprisebox-survey-checkout.js]: this is a temp remove need to added it back once the API is fixed.

Comment (PygmalionPolymorph) [reply theme/assets/surprisebox-survey-checkout.js]: Alright :)

**Pull request merged**